### PR TITLE
Plugin interface examples

### PR DESCRIPTION
**Pull request by twometresteve**
Angular, Express and Vue examples update for RC2, which changes the plugin interface.

### CODE REVIEW

**Comment by twometresteve** [comment]
> Are you planning to do restify, koa and react as part of this PR too?

Yes, as long as you're happy to receive follow-up commits.  I just had top break away from keyboard.

File: examples/express/app.js [hunk 1]
```diff
@@ -2,8 +2,10 @@ const Bugsnag = require('@bugsnag/js')
 const express = require('express')
 const { readFileSync } = require('fs')
 
-Bugsnag.start(process.env.BUGSNAG_API_KEY)
-Bugsnag.use(require('@bugsnag/plugin-express'))
+Bugsnag.start({
+  apiKey: process.env.BUGSNAG_API_KEY,
+  plugins: [require('@bugsnag/plugin-express')]
+})
```
Comment (tomlongridge) [inline examples/express/app.js]: Is there anything wrong with doing this?

```suggestion
const BugsnagExpressPlugin = require('@bugsnag/plugin-express')

Bugsnag.start({
  apiKey: process.env.BUGSNAG_API_KEY,
  plugins: [BugsnagExpressPlugin]
})
```

This maps more to the structure of the docs (with the requires/imports going first) and, to my mind, reads better.

Comment (twometresteve) [reply examples/express/app.js]: Done

File: UPGRADING.md [hunk 2]
```diff
@@ -25,7 +25,6 @@ You can choose to hold on to the `Client` returned by `Bugsnag.start()`, or not.
 - `Bugsnag.setContext()`, `Bugsnag.getContext()`
 - `Bugsnag.setUser()`, `Bugsnag.getUser()`
 - `Bugsnag.addMetadata()`, `Bugsnag.getMetadata()`, `Bugsnag.clearMetadata()`
-- `Bugsnag.use()`, `Bugsnag.getPlugin()`
```
Comment (tomlongridge) [inline UPGRADING.md]: `Bugsnag.getPlugin()` still exists

Comment (twometresteve) [reply UPGRADING.md]: Reinstated.

**Comment by twometresteve** [comment]
> All looks good, but there is a minor inconsistency to resolve:
> 
> `BugsnagVuePlugin` vs `BugsnagPluginReact`.
> 
> I prefer the latter as it matches the order of the import, e.g. `@bugsnag/plugin-x`, but as long as they are consistent I'm happy.

Now done.

**Pull request merged**